ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -0,0 +1,33 @@
+import ivy
+
+
+def argmax(input, dim=None, keepdim=False):
+    return ivy.argmax(input, axis=dim, keepdims=keepdim)
+
+
+def argmin(input, dim=None, keepdim=False):
+    return ivy.argmin(input, axis=dim, keepdims=keepdim)
+
+
+def amax(input, dim, keepdim=False, *, out=None):
```

Comment:
Please add `dim = None` to this too to capture the same behavior as torch's function. It should also be added to `amin`.